platform/linux-generic/odp_packet_io.c

```diff
@@ -1816,6 +1901,11 @@ int odp_pktio_capability(odp_pktio_t pktio, odp_pktio_capability_t *capa)
 		capa->lso.proto.ipv4             = 1;
 		capa->lso.proto.custom           = 1;
 		capa->lso.mod_op.add_segment_num = 1;
+
+		capa->config.pktout.bit.tx_compl_ena = 1;
```

Comment:
capa.config should not be filled in, as we are planning to remove it. mode_all under should tell to the application that tx_compl is supported.

Reply:
For now, this needs to be here as application signals the activation of the feature with the bit when calling `odp_pktio_config()` and config and capa needs to match.